apps/ocean-api/__tests__/controllers/rawtx/test.test.ts

```diff
@@ -0,0 +1,77 @@
+import { OceanApiTesting } from '../../../testing/OceanApiTesting'
+import { Elliptic } from '@defichain/jellyfish-crypto'
+import { ApiException } from '@defichain/ocean-api-client'
+
+const apiTesting = OceanApiTesting.create()
+const privA = Elliptic.fromPrivKey(Buffer.from('619c335025c7f4012e556c2a58b2506e30b8511b53ade95ea316fd8c3286feb9', 'hex'))
+const privB = Elliptic.fromPrivKey(Buffer.from('557c4bdff86e59015987c1c7f3328a1fb4c2177b5e834f09c8cd10fae51af93b', 'hex'))
+
+beforeAll(async () => {
+  await apiTesting.start()
+  await apiTesting.testing.container.waitForWalletCoinbaseMaturity()
+})
+
+afterAll(async () => {
+  await apiTesting.stop()
+})
+
+it('should accept valid txn', async () => {
```

Comment:
```suggestion
it('fund() - should accept valid txn', async () => {
```

Reply:
huh??
the test is for `rawtx.test()`

Reply:
My bad, updated the `suggestion`. I mean the test is not enclosed with a `describe()`, but nothing major.